app/views/home/index.html.haml

```diff
@@ -12,6 +12,11 @@
             %br
             = link_to "View more details & apply", event_path(@upcoming_event.first), class: "btn btn-primary btn-lg btn-hero"
 
+.container
+  .row
+    .col-md-12
+      = render 'layouts/messages'
```

Comment:
One of the failing specs (unsubscribing a member from emails I believe) was looking for the flash message on the page the user is redirected to, which is the homepage.

This appeared to only show when the flash was present, so shouldn't affect the layout of the homepage otherwise – if it does and I've missed it let me know and I can rework it.

Alternatively that test may not be for functionality that's still used(?) so that ^ might not be wanted at all :)